src/config.h

```diff
@@ -105,19 +105,21 @@
 
 /* Test for __builtin_prefetch()
  * Supported in LLVM since 2.9: https://releases.llvm.org/2.9/docs/ReleaseNotes.html
- * Supported in GCC since 3.1 but we use 4.9 given it's too old: https://gcc.gnu.org/gcc-3.1/changes.html. */
+ * Supported in GCC since 3.1 but we use 4.8 given it's too old: https://gcc.gnu.org/gcc-3.1/changes.html. */
 #if defined(__clang__) && (__clang_major__ > 2 || (__clang_major__ == 2 && __clang_minor__ >= 9))
 #define HAS_BUILTIN_PREFETCH 1
-#elif defined(__GNUC__) && (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 9))
+#elif defined(__GNUC__) && (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 8))
 #define HAS_BUILTIN_PREFETCH 1
 #else
 #define HAS_BUILTIN_PREFETCH 0
 #endif
 
 #if HAS_BUILTIN_PREFETCH
+#define redis_prefetch(addr)  __builtin_prefetch(addr)
```

Comment:
is it equivalent to redis_prefetch_read?  what about also adding `/* Read with high locality */` for this line.

Reply:
i remove this one, just use `redis_prefetch_read`